### DISCUSSION TITLE
How to configure webpack for React-native-web for RN 0.59

### DISCUSSION THREAD

**Discussion by ZChenzy** [software-mansion/react-native-gesture-handler#1200] [Q&A]
## Description
I am trying to add this library to an existing React-native-web project and is having no luck with webpack. Does anyone have a working webpack sample I can take inspiration from?

### Package versions

- React: 16.8.3
- React Native: 0.59.9
- React Native Gesture Handler:  1.3.0
- React Native Web: 0.2.2
- Webpack: 4.41.5

**Comment by jgonet** [comment]
Moving this issue to Discussions.
